### PR TITLE
Fixed input field

### PR DESCRIPTION
**Pull request by plxity**
I have verified that this pull request:
fixes: #1129 
* [x] has no linting errors (`npm run lint`)
* [x] is from a uniquely-named feature branch and has been rebased on top of the latest master. (If I was asked to make more changes, I have made sure to rebase onto master then too)
* [x] is descriptively named and links to an issue number, i.e. `Fixes #123`

### CODE REVIEW

**Comment by plxity** [comment]
@catarak Please review this :)

**Comment by catarak** [comment]
thanks for working on this! i noticed that if you input a decimal number into the input box, and then press the increase or decrease buttons, it gets super funky. i think we should prevent users from entering in decimal numbers, or when a user inputs a decimal value, either round or floor the value.

**Comment by plxity** [comment]
@catarak Yeah, You are right. I have fixed it, Please review it again :)

File: client/modules/IDE/components/Preferences.jsx [hunk 1]
```diff
@@ -151,10 +154,9 @@ class Preferences extends React.Component {
                   aria-atomic="true"
                   value={this.state.fontSize}
                   onChange={this.onFontInputChange}
+                  type="text"
                   ref={(element) => { this.fontSizeInput = element; }}
-                  onClick={() => {
-                    this.fontSizeInput.select();
-                  }}
+                  onClick={() => { this.fontSizeInput.select(); }}
```
Comment (catarak) [inline client/modules/IDE/components/Preferences.jsx]: could this this line just be
```jsx
onClick={this.fontSizeInput.select}
```

Comment (plxity) [reply client/modules/IDE/components/Preferences.jsx]: @catarak It shows an error if you do so!

Comment (catarak) [reply client/modules/IDE/components/Preferences.jsx]: oh oh oh it's because it's a `ref`, got it 😸

Comment (plxity) [reply client/modules/IDE/components/Preferences.jsx]: @catarak Yeah exactly!
Can this be merged now? :)

**Pull request merged**